cmd/template/fallback.cfg.sample

```diff
@@ -10,5 +10,4 @@ policy:
   ReservedResources:
     CPU: 750m
 logger:
-  Debug: resource-manager,cache,policy,resource-control
-  
\ No newline at end of file
+  Debug: resource-manager,cache,policy,resource-control
```

Comment:
There is something wrong with this file, github complains that "no newline at the of file", can you verify that there is a newline "\n" at the end.

Reply:
Probably that is related to your editor then. You can see the exact characters what the commit contains with this command:
```
git show HEAD~1 | od -A x -t x1z -v -c
```
The last character of the file should be \n in this case, and not an empty line (which means two \n).